app/views/pages/moving-guide/civilians.html.erb

```diff
@@ -1,11 +1,331 @@
+          <li>When you pack for an overseas move, keep an accordian folder in your carry on bag containing copies of important documents you might need when going through customs in your new host country.</li>
+        </ul>
+      <% end %>
+    </div>
+
+    <div id="pack-day" class="article-section">
+      <h4>Packing Day</h4>
+
+      <p>Your packing day and moving dates are not the same thing. On packing day, the moving company sends a team of packers to your home and puts all your items into boxes, usually as fast as possible. The day your boxed are loaded onto the truck for moving day could be several days later.</p>
+
+      <p>The packers are supposed to call ahead of time to inform you they will be coming out on certain dates and within a specified time period. It's critical that you are available during this time and if for some reason they fail to show during the window, notify your <%= link_to 'local personal property office', offices_path %> quickly.</p>
+
+      <div class="usa-grid-full">
+        <div class="usa-width-one-third">
+          <p>A team of packers will load just about anything and everything that they can fit into boxes, which means there's a risk that important items may get packed up unintentionally.</p>
+          <p><strong><em>How should I pack?</em></strong><br>
+          Pack your belongings with unpacking in mind. Where do you want these boxes to go in your new place? The master bedroom, living room, or kitchen? When you move <%= abbr_tag('oconus') %>, there can be a language barrier between you and your movers in the new host country.</p>
+        </div>
+        <div class="usa-width-two-thirds">
+          <%= render layout: 'shared/pro_tip' do %>
+            <strong>Pro-Tips:</strong>
+            <ul>
+              <li><strong>Physically separate</strong> anything you plan to take/keep with you from the rest of your belongings into a closet or your vehicle before the packers arrive. Clearly mark this area with a "Do Not Pack" sign.</li>
+              <li><strong>Invite friends and family</strong> to keep an eye on things during packing day. It can be really helpful if you can have someone shadowing the packers, making sure your inventory is accurate, potentially even taking photos of the inside of boxes as they are packed.</li>
+              <li><strong>Label your boxes</strong> with a small piece of duct tape, each room with its own color. Others have found it helpful to number each room when you arrive in your host country, so that your movers know that the master bedroom is 1, the kitchen 2, and the living room 3.</li>
+              <li><%= link_to 'View more tips', page_path('moving-guide/tips') %></li>
+            </ul>
+          <% end %>
+        </div>
+      </div>
+      <p>Bottom line, if at any point something doesn't feel right, you should call your local personal property office and ask for the assistance from an inspector/quality assurance personnel.</p>
```

Comment:
There are links to the office locator every other time this text appears; should it go here too?